activerecord/lib/active_record/explain.rb

```diff
@@ -16,23 +16,32 @@ def collecting_queries_for_explain # :nodoc:
 
     # Makes the adapter execute EXPLAIN for the tuples of queries and bindings.
     # Returns a formatted string ready to be logged.
-    def exec_explain(queries) # :nodoc:
-      str = queries.map do |sql, binds|
-        msg = +"EXPLAIN for: #{sql}"
-        unless binds.empty?
-          msg << " "
-          msg << binds.map { |attr| render_bind(attr) }.inspect
+    def exec_explain(queries, format = :text) # :nodoc:
```

Comment:
I was also thinking about to split this method to be just router for `exec_explain_text` and `exec_explain_json` methods. @rafaelfranca do you think that's good idea?

Reply:
I think so.

Reply:
done